src/tests/test_forcefield.py

```diff
@@ -16,33 +14,30 @@ class FFTests(object):
 
     def test_FF_yields_consistent_results(self):
         """Check whether multiple calls to FF yield the same result"""
-        self.logger.debug("\nChecking consistency of ForceField constructor\n")
-        self.assertEqual(forcefield.FF(self.options),forcefield.FF(self.options),
-        msg = "\nGot two different forcefields despite using the same options as input")
+        print("\nChecking consistency of ForceField constructor\n")
+        assert forcefield.FF(self.options) == forcefield.FF(self.options), "Got two different forcefields despite using the same options as input"
 
     def test_make_function_return_value(self):
         """Check that make() return value meets expectation"""
         pvals = self.ff.pvals0
 
-        self.logger.debug("Running forcefield.make() with zero vector should not change pvals... ")
+        print("Running forcefield.make() with zero vector should not change pvals... ")
```

Comment:
Can we continue to use a logger? Where would it be defined?

Reply:
The logger is now part of the `ForceBalanceTestCase` class definition (defined in `setup_class`). So we can revert things to `self.logger`

Reply:
Thanks. There is a logger in the main ForceBalance codes and I'd like to know how the test logger differs / is similar to the main one. On a side note I need a refresher on why `src/logger.py` was designed that way, and write some good docs to explain them.